spec/spacex/launches_spec.rb

```diff
@@ -208,6 +208,119 @@
     end
   end
 
+  context "#info('68')", vcr: { cassette_name: 'launches/68' } do
+    subject do
+      SPACEX::Launches.info('68')
+    end
+
+    it 'returns info for flight number 68' do
+      expect(subject).to eq(
+        'flight_number' => 68,
+        'mission_name' => 'Telstar 18V',
+        'mission_id' => [
+          'F4F83DE'
+        ],
+        'launch_year' => '2018',
+        'launch_date_unix' => 1_536_554_700,
+        'launch_date_utc' => '2018-09-10T04:45:00.000Z',
+        'launch_date_local' => '2018-09-10T00:45:00-04:00',
+        'is_tentative' => false,
+        'tentative_max_precision' => 'hour',
+        'rocket' => {
+          'rocket_id' => 'falcon9',
+          'rocket_name' => 'Falcon 9',
```

Comment:
We're trying to test here not only the response but how we can access this information via SPACEX gem. 
I would add at least another test in addition to this (doesn't need to have all the fields. but at least two or three.) Something to test like: 

```ruby
expect(subject.rocket.rocket_name).to eq('Falcon 9')
```